pkgs/applications/networking/listadmin/default.nix

```diff
@@ -0,0 +1,50 @@
+{ lib, stdenv, fetchurl, makeWrapper, perlPackages }:
+
+stdenv.mkDerivation rec {
+  pname = "listadmin";
+  version = "2.73";
+
+  src = fetchurl {
+    url = "mirror://sourceforge/project/listadmin/${version}/listadmin-${version}.tar.gz";
+    sha256 = "00333d65ygdbm1hqr4yp2j8vh1cgh3hyfm7iy9y1alf0p0f6aqac";
+  };
+
+  buildInputs = [ perlPackages.perl ];
+  nativeBuildInputs = [ makeWrapper ];
+
+  configurePhase = ''
+  '';
+
+  buildPhase = ''
```

Comment:
You can remove this

Reply:
Then it calls `make` and that does stuff that isn't needed and prints warning. I’ll add a comment.

Reply:
You can also `dontBuild = true` which should be more clear than setting an empty build phase